### DISCUSSION TITLE
Convert P8 back to P7 compatible with TSMuxer

### DISCUSSION THREAD

**Discussion by AVX22** [quietvoid/dovi_tool#187] [Q&A] [answered]
Hi, has there been any update on this conversion? 
I found a piece of code back in 2021 ("Convert P8 to P7 MEL") but i do not know if this is possible at the moment.
A lot of people want the files to be read as a BD structure in OPPO.
Thanks!

**Comment by quietvoid** [comment, marked as answer]
What you want is to use mode 1.
However it's only lossless if the RPU is from a profile 7 MEL source and converted using mode 2.

For the OPPO, you still need an enhancement layer and there is currently no way to create them.
